clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/provider/view/AmazonSubnetProvider.groovy

```diff
@@ -96,6 +106,7 @@ class AmazonSubnetProvider implements SubnetProvider<AmazonSubnet> {
         purpose = "${splits[1]} (${splits[0]})"
       }
     }
+    def accountId = accounts.find{ account -> account.name.equals(parts.account)}.accountId
```

Comment:
Can we get a miss here? If one account is removed and the clouddriver cache has stale data here?

Reply:
yeah actually that could happen

one thought would be to rework this and stick the accountId into the cached data when it is stored? (would eliminate the need to resolve it on the way back out)

Reply:
Reworked in https://github.com/spinnaker/clouddriver/pull/3101